### PR TITLE
add phase0 check for certain plots

### PR DESCRIPTION
**Pull request by imKuehlschrank**

### CODE REVIEW

File: dqmgui/style/SiPixelMapsRenderPlugin.cc [hunk 1]
```diff
@@ -31,9 +31,13 @@ class SiPixelMapsRenderPlugin : public DQMRenderPlugin
...
-      if ((o.name.find( "PixelPhase1/" ) != std::string::npos || o.name.find( "PixelPilot/" ) != std::string::npos  )
-        && (std::string(o.object->GetName()).find( "Coord" ) != std::string::npos )) {
-        return true;
+      if (o.name.find( "PixelPhase1/Phase1_MechanicalView" ) != std::string::npos || o.name.find( "PixelPhase1/Tracks" ) != std::string::npos  )
+      {
+        if(o.object && std::string(o.object->GetName()).find( "Coord" ) != std::string::npos )
+          	return true;
+        else
+        	return false;
+
```
Comment (vanbesien) [inline dqmgui/style/SiPixelMapsRenderPlugin.cc]: Guys, can we have a minimum of hygiene here?
Line 37 starts with 10 spaces + a tab
Line 39 starts with 8 spaces + a tab
Line 40 why an empty line?
Line 41-43 shows the best practice of using {} even for one-liners, why be so inconsistent?

File: dqmgui/style/SiPixelRenderPlugin.cc [hunk 2]
```diff
@@ -269,7 +269,9 @@ class SiPixelRenderPlugin : public DQMRenderPlugin
 
       TH2F* obj2 = dynamic_cast<TH2F*>( o.object );
 
-      if( o.name.find("Summary") != std::string::npos && o.name.find("_Error_") == std::string::npos) 
+      if( o.name.find("Summary") != std::string::npos && 
+      	  o.name.find("_Error_") == std::string::npos && 
+      	  o.name.find("PixelPhase1/") != std::string::npos)
```
Comment (vanbesien) [inline dqmgui/style/SiPixelRenderPlugin.cc]: Same here:
Line 272: Trailing space
Line 273 starts with 6 spaces, followed by a tab, followed by another 2 spaces + trailing space
Line 274 starts with 6 spaces, followed by a tab, followed by another 2 spaces + trailing space

**Comment by vanbesien** [comment]
Ok, much better now.
Let's continue the normal flow from this point.
Test on the test system and let us know if this can go to production.

**Comment by imKuehlschrank** [comment]
Yes, this can go to production.

**Comment by threus** [comment]
@imKuehlschrank please close this PR, it is superseded by #458 
Thanks.

**Pull request closed without merging**